AppDevAnalytics/Classes/Analytics.swift

```diff
@@ -6,18 +6,30 @@
 //  Copyright © 2019 Kevin Chan. All rights reserved.
 //
 
-import Crashlytics
+import FirebaseAnalytics
 
-class AppDevAnalytics {
+public protocol Event {
+    var name: String { get }
+    var parameters: [String: Any]? { get }
+}
+
+public extension Event {
+    var parameters: [String: Any]? {
+        nil
+    }
+}
+
+public class AppDevAnalytics {
 
```

Comment:
If you're not adding blank newlines after ever object/extension declaration and before its closing `}`, then Delete the newline.